config/routes.rb

```diff
@@ -36,6 +36,15 @@
   end
 
   get 'pages/about'
+  get 'pages/my_personal_calendar'
+  get 'pages/my_personal_calendar_members'
+  get 'pages/my_personal_calendar_tags'
+  get 'pages/home'
+  get 'pages/settings'
+  get 'pages/create_team_personal'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.